HW_6/hw_6.2.4.js

```diff
@@ -0,0 +1,18 @@
+//4. Написать функцию, которая принимает на вход слово и проверяет, является ли это слово палиндромом
+
+function Palindrome(word) {
```

Comment:
```suggestion
function palindrome(word) {
```

Function names start in lower case, upper case is used for `constructor functions`